lib/db.js

```diff
@@ -721,6 +766,16 @@ module.exports = function (
     return this.pool.post('/account/' + account.uid.toString('hex') + '/verifyEmail')
   }
 
+  DB.prototype.verifyTokens = function (tokenVerificationId, tokenData) {
```

Comment:
I suggest calling the second argument something like `accountData` rather than `tokenData`, as we're not expecting anything token-specific in there.